### PR TITLE
Format None, True, False and NotImplemented as literals

### PR DESCRIPTION
**Pull request by serhiy-storchaka**
----
📚 Documentation preview 📚: https://cpython-previews--118758.org.readthedocs.build/

### CODE REVIEW

File: Misc/NEWS.d/3.11.0b1.rst [hunk 1]
```diff
@@ -664,8 +664,8 @@ for :func:`os.fcopyfile` available in macOs.
 .. nonce: l1p7CJ
 .. section: Library
 
-For @dataclass, add weakref_slot. Default is False. If True, and if
-slots=True, add a slot named "__weakref__", which will allow instances to be
+For ``@dataclass``, add ``weakref_slot``. Default is ``False``. If true, and if
```
Comment (AlexWaygood) [inline Misc/NEWS.d/3.11.0b1.rst]:
```suggestion
For :func:`@dataclass <dataclasses.dataclass>`, add *weakref_slot*.
The new parameter defaults to ``False``. If true, and if
```

File: Misc/NEWS.d/3.12.0a1.rst [hunk 2]
```diff
@@ -3561,8 +3561,8 @@ with :func:`os.pidfd_open` in non-blocking mode. Patch by Kumar Aditya.
 .. nonce: mkYl5q
 .. section: Library
 
-Implement Enum __contains__ that returns True or False to replace the
-deprecated behaviour that would sometimes raise a TypeError.
+Implement ``Enum.__contains__`` that returns ``True`` or ``False`` to replace the
+deprecated behaviour that would sometimes raise a ``TypeError``.
```
Comment (AlexWaygood) [inline Misc/NEWS.d/3.12.0a1.rst]:
```suggestion
deprecated behaviour that would sometimes raise a :exc:`TypeError`.
```

File: Misc/NEWS.d/3.13.0a1.rst [hunk 3]
```diff
@@ -3436,7 +3436,7 @@ added support for this decorator. Patch by Alex Waygood.
 .. nonce: C1ahtk
 .. section: Library
 
-Make pydoc.doc catch bad module ImportError when output stream is not None.
+Make :func:`pydoc.doc` catch bad module ``ImportError`` when output stream is not ``None``.
```
Comment (AlexWaygood) [inline Misc/NEWS.d/3.13.0a1.rst]:
```suggestion
Make :func:`pydoc.doc` catch bad module :exc:`ImportError`
when output stream is not ``None``.
```

File: Misc/NEWS.d/3.13.0a2.rst [hunk 4]
```diff
@@ -1354,8 +1354,8 @@ crash encountered after the first :meth:`tkinter.Tk` instance is destroyed.
 .. section: IDLE
 
 Add docstrings to the IDLE debugger module. Fix two bugs: initialize
-Idb.botframe (should be in Bdb); in Idb.in_rpc_code, check whether
-prev_frame is None before trying to use it. Greatly expand test_debugger.
+``Idb.botframe`` (should be in Bdb); in ``Idb.in_rpc_code``, check whether
+``prev_frame is None`` before trying to use it. Greatly expand test_debugger.
```
Comment (AlexWaygood) [inline Misc/NEWS.d/3.13.0a2.rst]:
```suggestion
``prev_frame`` is ``None`` before trying to use it. Greatly expand test_debugger.
```

File: Misc/NEWS.d/3.12.0a1.rst (hunk 2 above)
Comment (rhettinger) [reply Misc/NEWS.d/3.12.0a1.rst]: I don't think there is any value in turning TypeError to a link in this context.

**Review by rhettinger — APPROVED** [review]

File: Doc/c-api/object.rst [hunk 5]
```diff
@@ -65,7 +65,7 @@ Object Protocol
 
    Properly handle returning :c:data:`Py_NotImplemented` from within a C
    function (that is, create a new :term:`strong reference`
-   to NotImplemented and return it).
+   to ``NotImplemented`` and return it).
```
Comment (JelleZijlstra) [inline Doc/c-api/object.rst]: Why not this?
```suggestion
   to :const:`NotImplemented` and return it).
```

File: Misc/NEWS.d/3.12.0a1.rst (hunk 2 above)
Comment (AlexWaygood) [reply Misc/NEWS.d/3.12.0a1.rst]: I generally prefer to assume close to no knowledge on the part of the reader when it comes to the CPython docs. But I don't have a strong opinion here.

Comment (gvanrossum) [reply Misc/NEWS.d/3.12.0a1.rst]: I think there's enough value to keep the link.

**Pull request merged**